CMakeLists.txt

```diff
@@ -87,6 +87,8 @@ find_package(${Qt} ${QtMinVersion} REQUIRED Core Network Gui Test ${QtExtraModul
 get_filename_component(Qt_Prefix "${${Qt}_DIR}/../../../.." ABSOLUTE)
 message(STATUS "Using Qt ${${Qt}_VERSION} at ${Qt_Prefix}")
 
+find_package(${Qt}Keychain REQUIRED)
```

Comment:
Looks like this needs a matching change in the configuration for GitHub Actions CI.

Reply:
The CI was correct, CMake was a bit wrong